lib/src/lib/table/components/table/table.component.html

```diff
@@ -128,6 +128,7 @@
         mat-row
         *matRowDef="let row; columns: displayedColumns; let i = dataIndex"
         [ngClass]="getRowClasses(row, i)"
+        [style.background-color]="getRowColor(row)"
```

Comment:
Does this always overwrite a background colour already applied via css? In Bits we are putting a css styling on the odd rows and would not want this to stop working.

Reply:
well it will only override if you use the `rowColor` otherwise nothing will happen